webpack.config.js

```diff
@@ -63,6 +63,7 @@ const nodeConfig = baseConfig.clone()
         },
         output: {
             libraryTarget: 'commonjs2',
```

Comment:
Webpack's documentation suggests moving to `library.type` instead. In `scratch-webpack-configuration` `library.type` is set to `umd2`. Wouldn't that be conflicting? Probably should do:
```suggestion
            library: {
                type: 'commonjs'
            }
```
Also some of the other fields are already part of the shared configuration

Reply:
Thank you, updated the config